src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -148,6 +150,9 @@ protected override void ConfigureOptions()
             this.Add("list-extensions", "List all extension points and the extensions for each.",
                 v => ListExtensions = v != null);
 
+            this.Add("encoding=", "Specifies the encoding to use for Console standard output.",
```

Comment:
I imagine "utf-8", "ascii", & "unicode" will be the most popular encodings. Is there a different name you would prefer for this option, ie `--stdout-encoding`?